WordPress/src/main/java/org/wordpress/android/localcontentmigration/LocalMigrationResult.kt

```diff
@@ -98,6 +100,11 @@ sealed class LocalMigrationState {
     sealed class Finished: LocalMigrationState() {
         object Successful: Finished()
         data class Failure(val error: LocalMigrationError): Finished()
-        object DeleteOnly: Finished()
+    }
+    sealed class SingleStep: LocalMigrationState(), Parcelable {
+        @Parcelize
+        object DeleteSingleStep: SingleStep()
+        @Parcelize
+        object UpdateWPSingleStep: SingleStep()
```

Comment:
I wonder if we could find a more suitable place for these states? It seems that updating WordPress is not really part of the data migration, and more like a preliminary step, perhaps deserving its own space to represent the associated states that may be involved :thinking: . 

My initial intention when introducing this class was to provide a means for the orchestrator to communicate a simplified state to the view model (i.e. without exposing the UI layer to the complexity that may be handled in an encapsulated way in the data migration).

Are these single steps more or less UI states? If so, I wonder if we could move them to the view model.. wdyt?

As an example, for the already merged `DeleteOnly` state, I was able to move this, simplifying the interaction between the view model and orchestrator here: https://github.com/wordpress-mobile/WordPress-Android/pull/17549/commits/121c49ebbebee36f365092babbd74d1da46c00f1